src/main/java/com/crowd/bean/ProjectResponse.java

```diff
@@ -53,18 +54,10 @@ public ProjectResponse(Project project) {
         percendDonate = project.percendDonate();
         fullNameUser = project.getUser().fullName();
         created = project.getDate();
+        simpleStatus = project.getStatus().getStatus();
```

Comment:
It's better to get this value dynamically. Now, if you set new `status` via setter method `simpleStatus` will remake the same.